business-wiki-frontend/pages/index.tsx

```diff
@@ -5,27 +5,28 @@ import buttonStyles from "../shared/styles/Button.module.scss";
 import Head from "next/head";
 import React, { FormEvent, useContext } from "react";
 import Input from "../shared/components/TextInput";
+import Footer from "../shared/components/Footer";
```

Comment:
Polecam zapoznać się z tym tematem tutaj i uporządkować importy komponentów(i nie tylko, tak samo można zrobić z helperami itp.).
https://blog.logrocket.com/using-barrel-exports-organize-react-components/

Reply:
Resolved in e1e3c018091be32f2690176e894a291f9184010b